### DISCUSSION TITLE
Could not update a Shop Receipt with receipt_id

### DISCUSSION THREAD

**Discussion by nikolajevs86** [etsy/open-api#1115] [Q&A]
Hi, we have got many failed requests to update the order tracking. For some reason when we send tracking information Etsy responds with status: 409 and the error: "Could not update a Shop Receipt with receipt_id".
Can you please elaborate on what error means?

**Comment by Gareth064** [comment]
This is what a 409 error means in the real world.

409 Conflict - HTTP | MDN (mozilla.org)<https://developer.mozilla.org/en-US/docs/web/http/status/409>

I assume Etsy are using it for the same thing.
As what is throwing it, you would have to ask Etsy via email.

Send them a message at ***@***.***

All responses from the API will include a Header with a Client request Id, or something like that. This is an ID they can use to look for your Api call at their end. If you can get that and include it in the email then that might help them help you quicker.


Kind Regards

Gareth Doherty



It’s nice to be important,

But it’s more important to be nice.



________________________________
From: P.Nik ***@***.***>
Sent: 19 September 2023 09:38
To: etsy/open-api ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [etsy/open-api] Could not update a Shop Receipt with receipt_id (Discussion #1115)


Hi, we have got many failed requests to update the order tracking. For some reason when we send tracking information Etsy responds with status: 409 and the error: "Could not update a Shop Receipt with receipt_id".
Can you please elaborate on what error means?
